Find a functionally equivalent code:android.app.ActivityThread.getTopLevelResources:COMMENT
Method Modifier: 
Comment:/**
 * Creates the top level resources for the given package.
 */

@@ -1,3 +1,3 @@
 {
-    return mResourcesManager.getTopLevelResources(resDir, displayId, overrideConfiguration, pkgInfo.getCompatibilityInfo(), null);
+    return mResourcesManager.getTopLevelResources(resDir, splitResDirs, overlayDirs, libDirs, displayId, overrideConfiguration, pkgInfo.getCompatibilityInfo(), null);
 }

